layouts/default.vue

```diff
@@ -59,6 +86,11 @@
     </v-main>
     <v-footer app>
       <span>&copy; 2020</span>
+      <h5 :class="`text-md-h6`" class="ml-3" style="font-weight:normal">
```

Comment:
```suggestion
      <h5 class="text-md-h6 ml-3" style="font-weight:normal">
```

Combine to one `class` does not having data binding.